tests/static_map/rehash_test.cu

```diff
@@ -36,14 +34,11 @@ TEST_CASE("Rehash", "")
                                      cuco::empty_value<mapped_type>{-1},
                                      cuco::erased_key<key_type>{-2}};
 
-  thrust::device_vector<key_type> d_keys(num_keys);
-  thrust::device_vector<mapped_type> d_values(num_keys);
+  auto keys_begin = thrust::counting_iterator<key_type>(1);
 
-  thrust::sequence(d_keys.begin(), d_keys.end());
-  thrust::sequence(d_values.begin(), d_values.end());
-
-  auto pairs_begin =
-    thrust::make_zip_iterator(thrust::make_tuple(d_keys.begin(), d_values.begin()));
+  auto pairs_begin = thrust::make_transform_iterator(keys_begin, [] __device__(key_type const& x) {
```

Comment:
`proclaim_return_type`

Reply:
Addressed in [6cc9040](https://github.com/NVIDIA/cuCollections/pull/387/commits/6cc9040acc17e1873d5b3976ac07438a7da3b644)